app/screens/conversion-flow/conversion-confirmation-screen.tsx

```diff
@@ -269,7 +276,17 @@ export const ConversionConfirmationScreen: React.FC<Props> = ({ route }) => {
         <View style={styles.conversionInfoField}>
           <Text style={styles.conversionInfoFieldTitle}>{LL.common.rate()}</Text>
           <Text style={styles.conversionInfoFieldValue}>
-            ~ {formatMoneyAmount(usdPerBtc)} / 1 BTC
+            ~{" "}
+            {formatMoneyAmount(
+              convertMoneyAmount(
+                {
+                  amount: 1 * 100000000,
```

Comment:
```suggestion
                  amount: 1 * 100_000_000,
```

Reply:
still have the issue today? a lot of things were not necessarily properly set up on ci. like we were using node12 not long enough. I would give it another try. I think it really helps on readability.

Reply:
seems it was introduce in ES12

<img width="798" alt="image" src="https://user-images.githubusercontent.com/11156370/222266500-a92da909-a762-4b92-8cb7-73b45a7517b5.png">